sormas-api/src/main/resources/strings.properties

```diff
@@ -595,6 +595,8 @@ messageFollowUpCanceled = Follow-up of all selected contacts has been canceled
 messageFollowUpStatusChanged = Follow-up of all selected contacts that have follow-up has been set to lost to follow-up
 messageFacilityChanged = You have changed the facility of this case. Do you want to transfer the case to the new facility (the hospitalization may be updated) or do you only want to edit the data to correct a mistake?
 messageFacilityMulitChanged = You have changed the facility of multiple cases. Do you want to transfer these cases to the new facility (the hospitalization may be updated) or do you only want to edit their data to correct mistakes?
+messageGdpr = In accordance with the principle of data minimization imposed by the GDPR, you must ensure that you enter here only objective data absolutely necessary for the processing of the file. In particular, you must not enter genetic or biometric data, philosophical, political, religious, trade union opinions, sexual life or orientation, racial or ethnic origin, offenses, convictions, security measures, criminal record ... or any other data that does not would not be related to the health surveyIn accordance with the principle of data minimization imposed by the GDPR, you must ensure that you enter here only objective data absolutely necessary for the processing of the file. In particular, you must not enter genetic or biometric data, philosophical, political, religious, trade union opinions, sexual life or orientation, racial or ethnic origin, offenses, convictions, security measures, criminal record ... or any other data that does not would not be related to the health survey.
```

Comment:
You've accidentally duplicated the text here - also, I'd like to ask you to remove the term "racial" because it's a scientifically refuted concept.